### DISCUSSION TITLE
docker安装运行之后，选择节点点击启动报错：failed to start v2ray-core:

### DISCUSSION THREAD

**Discussion by ishuliang** [v2rayA/v2rayA#1408] [Q&A]
![image](https://github.com/v2rayA/v2rayA/assets/105412844/ee578681-0c6e-41cb-927d-4d11ac499885)

**Comment by helongjk** [comment]
没有权限，把权限打开就好了

**Reply by cy28438981** [reply]
> 没有权限，把权限打开就好了

大哥你好，我是爱快路由中的docker安装了的v2rayA，想要打开权限，如何打开？

**Comment by aceslup** [comment]
他指的应该是这个 --privileged 参数
